src/features/instance/operations/schemas/addCSVDataFormSchema.ts

```diff
@@ -0,0 +1,13 @@
+import { z } from 'zod';
+
+export const AddCSVDataFormSchema = z.object({
+  fileData: z
+    .any(),
```

Comment:
You can likely touch up this type now. It probably isn't a file, but it's something.